projects/packages/my-jetpack/changelog/master-merge-conflict-resolve

```diff
@@ -0,0 +1,3 @@
+Significance: patch
+Type: compat
```

Comment:
Where we've changed the use of `getJetpackData` that should probably be what we have in the changelog. It's maybe `changed` as a type?

Reply:
Master merge had some conflicts in this folder which I had to resolve. While pushing, it asked me to add a changelog file. It doesn't have anything to do with `getJetpackData`